kube/services/fence/fence-deploy.yaml

```diff
@@ -46,6 +46,8 @@ spec:
 # DEPRECATED! Remove when all commons are no longer using local_settings.py
 #             for fence.
 # -----------------------------------------------------------------------------
+        - name: logs-folder
```

Comment:
Should we add pormetheus annotations to the pod?
   https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/ambassador/ambassador-deploy.yaml#L17

Reply:
Oh wow, this is amazing! Thanks for the suggestion.